ipatests/test_ipalib/test_config.py

```diff
@@ -562,6 +562,7 @@ def test_finalize_core(self):
 
         # Test using DEFAULT_CONFIG:
         defaults = dict(constants.DEFAULT_CONFIG)
+        defaults['fips_mode'] = object
```

Comment:
Yes, to skip the value check (few lines below - https://github.com/freeipa/freeipa/pull/437/files/28c57e48423e77756eb152068db44933cd0540f7#diff-c5fcc8900faeb561ad5f2a58fef94c85L568 ). If we set it to `None`, the test will check whether `fips_mode` value is `None` - which is not the desired behavior.